### ISSUE TITLE
No items

### ISSUE THREAD

**Issue by Krevik** [SirPlease/L4D2-Competitive-Rework#196]
Sometimes the game loads with no items spawned (even these starting like medkits, shotgun, smg/uzi etc).
Mode: versus
Matchmode: no matchmode selected

**Comment by SirPlease** [comment]
I'll attempt to verify this, I did make an earlier issue where it doesn't spawn at all On The Passing finale, but this was with a matchmode loaded.

**Comment by SirPlease** [comment]
I haven't been able to reproduce this bug, are there maps in particular you're having this issue on?

**Comment by Krevik** [comment]
> ce this bug, are there maps in particular y

It happened to me on "The parish" and "No mercy"

**Comment by Krevik** [comment]
Again - this time on "Death Toll" - no medkits. Nothing weird in logs.

Also happend to me today on my second almost clear server - no medkits on "Dead Air". The only things I edited on this server are admins and server.cfg (rates, etc). I've got there only sourcemod and metamod. Can it be source mod / meta mod version problem? On that server I use 1.10.6 source and 1.7.2 meta. On Competitive server I use the version you supported in this git repo.
